applications/protocols_serialization/client/snippets/debug/debug.conf

```diff
@@ -20,7 +20,8 @@ CONFIG_LOG_MODE_DEFERRED=y
 CONFIG_LOG_MAX_LEVEL=4
 
 CONFIG_LOG_BACKEND_RTT=n
-CONFIG_SHELL_LOG_BACKEND=y
+#CONFIG_SHELL_LOG_BACKEND=y
```

Comment:
Why this change? I guess if both logger and shell write to UART, we may expect some interwoven lines and that's why you typically use shell logging backend when shell is enabled.

Reply:
I can change it to a comment. But as it turns out when we are processing the shell command the logs aren't printed out since shell is busy. moving it to UART backend breaks formatting a bit but do not delay logging.